src/template.js

```diff
@@ -14,10 +14,15 @@ const thumbnailFileExtension = ".png"
   if (imageKey) {
     return imageKey;
   }
-  let parsedTemplateName = templateName
+  
+  let resolvedTemplateName = templateName
     ? templateName.toLowerCase().trim()
     : `no-key`;
-  return parsedTemplateName.replace(/\s+/g, "-");
+
+  let templateName_NoSlashes = resolvedTemplateName.replace(/\//g, "-")
+  let templateName_NoSpaces = templateName_NoSlashes.replace(/\s+/g, "-");
+
+  return templateName_NoSpaces
```

Comment:
NAB: I would probably just shorten this all down to
```js
return templateName?.toLowerCase().trim().replace(/\/|\s+/g, "-") || "no-key"
```
Making use of a regex `or` character `|` there to do it one regex. Obviously doesn't affect functionality but may as well reduce code where we can!

Reply:
Yep, that makes perfect sense. I'll get that changed.